README.md

```diff
@@ -98,4 +105,79 @@ automatically remove set all auto-merge labels from the PR
 _Note_: at the moment the Dlang-Bot doesn't store _any_ authentication token, so
 it can't perform actions on behalf of others.
 
-<a name="canceling-state" />
+<a name="warn-common-mistakes" />
```

Comment:
> Doesn't GitHub automatically create linkable anchors for headings? It even shows the link thing in the margin when you hover over it.

It does, but it's rather annoying to keep it in sync. For example:

https://github.com/dlang-bots/dlang-bot#nerdy-details-1

The manual way here has the advantage that one doesn't break links by accident.